contents/scala/Scala.md

```diff
@@ -0,0 +1,241 @@
+<frontmatter>
+  title: Introduction to Scala
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Scala
+
+Author: [Wang Chao](https://github.com/fzdy1914)
+
+**Table of Contents**
+
+* [Overview](#overview)
+  * [Characteristic of Scala](#characteristic-of-scala)
+    * [Statically Typed](#statically-typed)
+    * [Object Oriented](#object-oriented)
+    * [Functional](#functional)
+* [Getting Started](#getting-started)
+* [Why Scala](#why-scala)
+  * [Operator Overloading](#operator-overloading)
+  * [Mixed-In Trait](#mixed-in-trait)
+  * [Type Enrichment](#type-enrichment)
+  * [Pattern Matching](#pattern-matching)
+
+# Overview
+
+**Scala** is a modern *multi-paradigm* programming language. It designed to integrate features of 
+*object-oriented* and *functional* languages in a *concise*, *elegant*, and *type-safe* way.
+
+## Characteristic of Scala
```

Comment:
Could this part be merged in "Why scala" section? It seems that both of them are Scala features.

Reply:
I would like to categorize them as Characteristics instead of features. From my perspective, characteristics are more general and feature are more specific.